runtime/doc/repeat.txt

```diff
@@ -762,6 +765,35 @@ the command after changing the plugin help: >
 	:helptags path/start/foobar/doc
 	:helptags path/opt/fooextra/doc
 
+The messages that are in the lang/<lang_id>/LC_MESSAGES/foo.po file need to be
+translated to a format that the |gettext()| function understands by running the
+*msgfmt* program. This will result in a lang/<lang_id>/LC_MESSAGES/foo.mo
+file.
+
+In your plugin, you need to call the |bindtextdomain()| function as follows.
+This assumes that the directory structure is as above: >
+	:call bindtextdomain("foo", getcwd()~"../lang")
+<
+You only need to do this once. After this call, you can use: >
+	:echo gettext("Hello", "foo")
+>
+to get the text "Hello" translated to the user's preferred language (if the
+plugin messages have been translated to this language).
+
+If you need to use the message in a different encoding from the current one,
+you can use (for instance) >
+	:echo gettext("Hello", "foo", "utf-8")
+<or >
+	:echo gettext("Hello", "foo", "cp1251")
```

Comment:
so the returned encoding is in `cp1251`?

Reply:
Yes, it is.